systemd/tinc@.service.in

```diff
@@ -4,6 +4,9 @@ Documentation=info:tinc
 Documentation=man:tinc(8) man:tinc.conf(5)
 Documentation=http://tinc-vpn.org/docs/
 PartOf=tinc.service
+Before=systemd-user-sessions.service
+After=network-online.target
+Wants=network-online.target
```

Comment:
Not in favor of this, see: https://github.com/gsliepen/tinc/pull/326#discussion_r711580446